load-testing/load-test-descriptor.json

```diff
@@ -0,0 +1,132 @@
+    { "duration": "1s", "target": 21 },
+    { "duration": "20s", "target": 21 },
+    { "duration": "15s", "target": 21 },
+    { "duration": "1s", "target": 22 },
+    { "duration": "20s", "target": 22 },
+    { "duration": "15s", "target": 22 },
+    { "duration": "1s", "target": 23 },
+    { "duration": "20s", "target": 23 },
+    { "duration": "15s", "target": 23 },
+    { "duration": "1s", "target": 24 },
+    { "duration": "20s", "target": 24 },
+    { "duration": "15s", "target": 24 },
+    { "duration": "1s", "target": 25 },
+    { "duration": "20s", "target": 25 },
+    { "duration": "15s", "target": 25 },
+    { "duration": "1s", "target": 26 },
+    { "duration": "20s", "target": 26 },
+    { "duration": "15s", "target": 26 },
+    { "duration": "1s", "target": 27 },
+    { "duration": "20s", "target": 27 },
+    { "duration": "15s", "target": 27 },
+    { "duration": "1s", "target": 28 },
+    { "duration": "20s", "target": 28 },
+    { "duration": "15s", "target": 28 },
+    { "duration": "1s", "target": 29 },
+    { "duration": "20s", "target": 29 },
+    { "duration": "15s", "target": 29 },
+    { "duration": "1s", "target": 30 },
+    { "duration": "20s", "target": 30 },
+    { "duration": "15s", "target": 30 },
+    { "duration": "1s", "target": 31 },
+    { "duration": "15s", "target": 31 },
+    { "duration": "1s", "target": 32 },
+    { "duration": "15s", "target": 32 },
+    { "duration": "1s", "target": 33 },
+    { "duration": "15s", "target": 33 },
+    { "duration": "1s", "target": 34 },
+    { "duration": "15s", "target": 34 },
+    { "duration": "1s", "target": 35 },
+    { "duration": "15s", "target": 35 },
+    { "duration": "1s", "target": 36 },
+    { "duration": "15s", "target": 36 },
+    { "duration": "1s", "target": 37 },
+    { "duration": "15s", "target": 37 },
+    { "duration": "1s", "target": 38 },
+    { "duration": "15s", "target": 38 },
+    { "duration": "1s", "target": 39 },
+    { "duration": "15s", "target": 39 },
+    { "duration": "1s", "target": 40 },
+    { "duration": "15s", "target": 40 },
+    { "duration": "1s", "target": 40 },
+    { "duration": "15s", "target": 40 },
+    { "duration": "1s", "target": 41 },
+    { "duration": "15s", "target": 41 },
+    { "duration": "1s", "target": 42 },
+    { "duration": "15s", "target": 42 },
+    { "duration": "1s", "target": 43 },
+    { "duration": "15s", "target": 43 },
+    { "duration": "1s", "target": 44 },
+    { "duration": "15s", "target": 44 },
+    { "duration": "1s", "target": 45 },
+    { "duration": "15s", "target": 45 },
+    { "duration": "1s", "target": 46 },
+    { "duration": "15s", "target": 46 },
+    { "duration": "1s", "target": 47 },
+    { "duration": "15s", "target": 47 },
+    { "duration": "1s", "target": 48 },
+    { "duration": "15s", "target": 48 },
+    { "duration": "1s", "target": 49 },
+    { "duration": "15s", "target": 49 },
+    { "duration": "1s", "target": 50 },
+    { "duration": "120s", "target": 50 }
```

Comment:
we should come up with better scenarios to perform load testing! Load on a particular endpoint can be gradually increased with time and RPS to get bet out of the services. maybe some spike testing or stress testing!

Reply:
Feel free to add more sample scenarios in a new PR